examples/laser/analysis_laser_init_chirp.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python3
+
+# Copyright 2024
+#
+# This file is part of HiPACE++.
+#
+# Authors: Xingjian Hui
+# License: BSD-3-Clause-LBNL
+
+import argparse
+import numpy as np
+import scipy.constants as scc
+from openpmd_viewer.addons import LpaDiagnostics
+
+def get_zeta(Ar, m, w0, L):
+    nu = 0
+    summ = 0
+    laser_module = np.abs(Ar)
+    phi_envelop = np.array(np.arctan2(Ar.imag, Ar.real))
+    # unwrap phi_envelop
+    phi_envelop = np.unwrap(np.unwrap(phi_envelop, axis=0), axis=1)
+    # calculate pphi_pz
+    z_diff = np.diff(m.z)
+    x_diff = np.diff(m.x)
+    pphi_pz = (np.diff(phi_envelop, axis=0)).T / (z_diff / scc.c)
+    pphi_pzpy = (np.diff(pphi_pz, axis=0)).T / x_diff
```

Comment:
If used only once, no need to define a variable.
```suggestion
    # calculate pphi_pz
    pphi_pz = (np.diff(phi_envelop, axis=0)).T / (np.diff(m.z) / scc.c)
    pphi_pzpy = (np.diff(pphi_pz, axis=0)).T / np.diff(m.x)
```